macaroonbakery/bakery.py

```diff
@@ -209,3 +221,14 @@ def _extract_urls(response):
     visit_url = response_json['Info']['VisitURL']
     wait_url = response_json['Info']['WaitURL']
     return visit_url, wait_url
+
+
+class ThirdPartyInfo:
+    def __init__(self, version, public_key):
```

Comment:
Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings